### ISSUE TITLE
[ENHANCEMENT] Keyboard pop-up animation -> App Drawer

### ISSUE THREAD

**Issue by direttorato-s** [Deletescape-Media/Lawnchair#3977]
### Describe the feature

1/ Keyboard pop-up that follow app drawer animation like Pixel Launcher style - Best solution

2/ Keyboard pop-up too delayed - Current animation

### Why would it be useful to add?

It is visually better.

### Did the feature exist in Lawnchair v2? (Play Store version)

No

### Example(s)

1/

https://github.com/LawnchairLauncher/lawnchair/assets/98133370/9b37695c-917c-48a9-9034-84123e996995

2/

https://github.com/LawnchairLauncher/lawnchair/assets/98133370/69314d6c-4b0c-4268-a107-fb12ad5f7127

**Comment by AndrewDallas** [comment]
This would be a very nice little addition.

**Comment by pablo03v** [comment]
Would be nice and also would be cool to see the keyboard popup every time

**Comment by alexandrius** [comment]
Can we at least have 0 delay to open keyboard? If the "Automatically show keyboard" is enabled it's fast and works perfectly for Swipe Up gesture.

**Comment by Zk2u** [comment]
Yes please. The easiest way to do this would just to be focusing the keyboard at the start of the animation vs after it's finished.

**Comment by SuperDragonXD** [comment]
Closing this issue as this has been resolved in the `16-dev` experimental branch.
